CONTRIBUTING.md

```diff
@@ -43,31 +43,52 @@ button on any of our documentation pages.
 
 ## Contributing via Pull Requests
 
-Contributions via Pull Requests (PRs) are much appreciated. Before sending us a
-pull request, please ensure that:
+We appreciate your contributions! Here’s a breakdown of the steps based on your relationship with the owner SignalFX organization:
 
-1. You are working against the latest source on the `main` branch.
-2. You check existing open, and recently merged, pull requests to make sure
-   someone else hasn't addressed the problem already.
-3. You open an issue to discuss any significant work - we would hate for your
-   time to be wasted.
-4. You submit PRs that are easy to review and ideally less 500 lines of code.
-   Multiple PRs can be submitted for larger contributions.
+### Case 1: Public Contributors (No Connection to SignalFX Organization)
+
+As a **public contributor** with no relation to SignalFX organization, you will likely **not have permission to create a feature branch** in the SignalFX repository. Here’s what to do:
```

Comment:
```suggestion
As a **public contributor** with no relation to Signalfx github organization, you will likely **not have permission to create a feature branch** in this repository. Here’s what to do:
```
We are referring to the github org here right? I think we should use Cisco/Splunk other wise

Reply:
- I mean the signalfx github org, I'm trying to be concise with wording here and elsewhere. Let me take another stab at it.
- I would like to hold off on introducing "Cisco" language until a separate PR where we can update the documentation in the whole repo in a single PR.